packages/eslint-config-terra/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Breaking
+  * Update to eslint ^7.19.0.
+
 * Fixed
   * Resolved missing peer dependency warning by locking into a previous version of eslint-config-airbnb.
```

Comment:
Do we want to release this before the breaking changes?

Reply:
👍 Yes, thats one of the reasons this is marked as 'do not merge'